packages/itwinui-css/backstop/tests/tabs.html

```diff
@@ -85,7 +86,8 @@ <h2>Default</h2>
                 aria-hidden="true"
               ></svg-placeholder>
               <span class="iui-tab-label">
-                <div>Left</div>
+                <div>This is a very long 'Left' tab content, which hopefully will wrap if all tab headers content is
+                  pretty long and cannot fit in viewport width</div>
                 <div class="iui-tab-description">Sublabel</div>
```

Comment:
Maybe test a long sublabel instead of main label for the 2nd tab?

Reply:
added